SpatialGDK/Source/Public/Schema/Component.h

```diff
@@ -2,9 +2,14 @@
 
 #pragma once
 
-struct SpatialComponent
+namespace improbable
```

Comment:
Should we add a comment to coding standards about namespace tabbings?

Reply:
Yes I wasn't sure which tabbing we preferred, I think that is a minor detail though compared to having the namespaces at all

Reply:
@improbable-valentyn @joshuahuburn  what do you think?

Reply:
I think that this current style is fine. It's how we did it at MS.